### PR TITLE
Add implementation of MVVM in Javascript

### PR DESCRIPTION
**Pull request by ting-dev-coder**
## Status
Done

## Description
Add implementation / description of model view viewmodel (MVVM) #15 
I haven't written anything for the README.md as it seems to be included in another issue, #25.

## Breaking Changes
No

## Type of Change
📝 Code

### CODE REVIEW

File: patterns/Architectual/Model_View_ViewModel/style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#mvvm {
+  text-align: center;
+  margin-top: 50px;
+}
+#mvvm h1 {
+  padding-bottom: 64px;
+}
+
+#mvvm button {
+  padding: 15px 30px; /* 按钮大一些 */
```
Comment (ash-teach) [inline patterns/Architectual/Model_View_ViewModel/style.css]: Can you translate this comment and the others to English? Or remove if they're not necessary.

File: patterns/Architectual/Model_View_ViewModel/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+</head>
+<body>
+	<div id="mvvm">
+		<h1>A Sample MVVM Implement</h1>
+    <button v-click="addNumber">+</button>
+		<p>{{number}}</p>
+		<button v-click="minusNumber">-</button>
+	</div>
+
+  <script type="module">
```
Comment (ash-teach) [inline patterns/Architectual/Model_View_ViewModel/index.html]: Can you fix the indentation on lines 12 and 17? Will make it easier to read.

**Review by ash-teach — CHANGES_REQUESTED** [review]

**Comment by ash-teach** [comment]
Great work, @ting-dev-coder. I've requested some very minor changes; however, I think more could be done on this documentation-wise. You've used a lot of clever and modern techniques that won't be familiar to everyone (I've been out of the JS ecosystem for a few year, so there are a few things that I didn't recognize right off!)

I think once you make those minor changes we'll merge, but then I will open a few more issues relating to creating documentation. Since there is a lot going on in this implementation, it might give some folks who haven't had a chance to contribute an opportunity to help out.

File: patterns/Architectual/Model_View_ViewModel/style.css (hunk 1 above)
Comment (ting-dev-coder) [reply patterns/Architectual/Model_View_ViewModel/style.css]: sure , they were used for todo notes to me 😂 so they are not necessary

**Comment by ting-dev-coder** [comment]
@ash-teach 
Done with modifications based on the above suggestions!
I agree! Some of them require some time to connect each logic, and that's why I feel like reading source code could give a boost to a developer!

**Pull request merged**